dotCMS/src/integration-test/java/com/dotcms/concurrent/scheduler/TestSynchronizedRunnable.java

```diff
@@ -0,0 +1,39 @@
+package com.dotcms.concurrent.scheduler;
+
+import com.dotmarketing.exception.DotRuntimeException;
+import io.vavr.control.Try;
+
+import java.io.Serializable;
+
+
+public class TestSynchronizedRunnable implements Runnable, Serializable {
```

Comment:
Codacy found an issue: [Classes implementing Serializable should set a serialVersionUID](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6006494)